libqtile/window.py

```diff
@@ -321,9 +391,25 @@ def update_state(self):
         if self.qtile.config.auto_fullscreen:
             triggered.append('fullscreen')
 
+        # https://specifications.freedesktop.org/wm-spec/1.3/ar01s07.html#STACKINGORDER
         state = self.window.get_net_wm_state()
+        _type = self.window.get_wm_type()
+        logger.debug('_NET_WM_STATE (%s): %s', self.name, state)
+        logger.debug('_NET_WM_WINDOW_TYPE (%s): %s', self.name, _type)
+        old = self.z.wm
+        if _type == 'desktop':
+            self.z.wm = DESKTOP_LAYER
```

Comment:
Might be nice to add some tests for this behavior.

Reply:
I'll have a look.